airbyte-ci/connectors/pipelines/pipelines/helpers/utils.py

```diff
@@ -28,7 +28,7 @@
     from pipelines.airbyte_ci.connectors.context import ConnectorContext
 
 DAGGER_CONFIG = Config(log_output=sys.stderr)
-AIRBYTE_REPO_URL = "https://github.com/airbytehq/airbyte.git"
+AIRBYTE_REPO_URL = os.environ.get("AIRBYTE_REPO_URL", "https://github.com/airbytehq/airbyte.git")
```

Comment:
Funny, I thought this variable was set in the run-airbyte-ci GHA... was it just not being used?

Reply:
We can be better at naming variables here :) 

The `CI_GIT_REPO_URL` is used for forks. We diff `airbyte` repo (with the static `AIRBYTE_REPO_URL` const`) and the fork (`CI_GIT_REPO_URL`. The fork has the same exact structure as the main `airbyte`, but your repo, with the submodule is slightly different, so the root repo (`AIRBYTE_REPO_URL`) has to be set to your repo private repo for diffing.